src/region_replicate_music.lua

```diff
@@ -0,0 +1,32 @@
+function plugindef()
+    finaleplugin.Author = "Michael McClennan"
+    finaleplugin.Version = 1.0
+    finaleplugin.Copyright = "2022/01/03"
+    finaleplugin.HandlesUndo = true
+return "Replicate Music", "Replicate Music", "Inspired by the 'r' key in Sibelius, this script copies the selected music, and pastes it directly to the right"
+end
+
+--[[
+Inspired by the 'r' key in Sibelius, this script copies the selected music, and pastes it directly to the right.
+Works with a single or multiple measures.
+When activated with a shortcut or hotkey, ultra fast replication is possible.
+--]]
```

Comment:
Could we move this to the `finaleplugin.Notes` field so it shows up on the website?

When that's done, LGTM and I'll merge.